python/aswfdocker/tests/test_releaser.py

```diff
@@ -5,9 +5,6 @@
 import unittest
 from unittest import mock
 import logging
-import tempfile
```

Comment:
Thanks! I really thought my pylint setup would have caught that... I need to verify why it got through!